eng/test-job.yml

```diff
@@ -179,6 +181,12 @@ jobs:
           # DotNet-HelixApi-Access variable group
           helixAccessToken: $(HelixApiAccessToken)
 
+        # Sets up the template to run the corefx tests
```

Comment:
I don't think we should use test-job.yml file for running corefx tests.

This file also contains logic for building coreclr tests and I don't want this logic intermingles with corefx tests submission logic.

I would prefer to have a separate file (corefx-test-job.yml?) that just calls eng/send-to-helix-step.yml with the specified helixProjectFile

Reply:
I think this makes more sense as well.

Reply:
There would be a lot duplication. Small things like some of the variables and the DotNet-HelixApi-Access.  Bigger things like part of build-test scripts is putting together the "testhost" (and core_root which I don't need) which is the the corefx/coreclr bits to test in a SDK "install" like layout.  And that relies on downloading the "product build" step.

The build-test scripts do have options to just generate the core_root/testhost directories (buildtesthostonly in build-test.cmd and generatetesthostonly in build-test.sh) so it would be better not to have to build the coreclr tests when running the corefx.

I'm not sure I have time to make this big of change (and getting working) in the day and a half before I go on vacation for 3 weeks.

Reply:
I also realized that if we stayed with one test-job.yml for both, there would need to be more conditions in the "scenarios" section.  For now just "normal" should be the corefx test scenarios, but that may change in the future.

Reply:
In the end, I tried putting together a corefx-test-job.yml and other than the build-test script options I mentioned above, and the helixProjectFile variable, there are not significant differences.

Reply:
We should avoid building coreclr tests if we don't need to in the corefx job. It seems like it would be worth introducing another template with the common logic shared between corefx and cocreclr test jobs.